docs/src/components/InteractiveExample/index.tsx

```diff
@@ -15,6 +15,56 @@ import ResetDark from '@site/static/img/reset-dark.svg';
 
 import styles from './styles.module.css';
 
+import ts from 'typescript';
+import prettier from 'prettier/standalone';
+import babelParser from 'prettier/parser-babel';
+
+function compileTSXtoJSX(tsxCode: string) {
+  const TEXT_TO_REPLACE_1 = '// 1-COMMENT-TO-REPLACE';
+  const TEXT_TO_REPLACE_2 = '// 2-COMMENT-TO-REPLACE';
```

Comment:
This is unclear for me. Could you explain me what is the purpose of this comments?

Reply:
Yes. The purpose of these comments is to maintain blank spaces between lines. So in my opinion it should stay there.